crawl-ref/source/mon-spell.h

```diff
@@ -1412,6 +1412,15 @@ static const mon_spellbook mspell_list[] =
       }
     },
 
+    {  MST_MLIOGLOTL,
+      {
+       { SPELL_MIGHT, 8, MON_SPELL_NATURAL },
+       { SPELL_CAUSE_FEAR, 10, MON_SPELL_MAGICAL | MON_SPELL_EMERGENCY },
+       { SPELL_BLINK, 5, MON_SPELL_MAGICAL | MON_SPELL_EMERGENCY },
```

Comment:
This seems quite annoying. Are you sure you want it?

Reply:
From some more playtesting I'm inclined to agree and have removed blink from Mlio